### ISSUE TITLE
question about datasets

### ISSUE THREAD

**Issue by margonotmango** [carla-simulator/imitation-learning#44]
hi,
I found that the value of the throttle in the dataset is basically 0.5 or 1, but the speed is basically uniform. Why? Assuming the throttle is always 0.5, the speed should always increase. I'm so confused.
Looking forward to your reply~

**Comment by AliBaheri** [comment]
@felipecode,
I have the same question here. Would it be possible to give some information in this regard?

**Comment by felipecode** [comment]
Throttle 0.5 is like a minimum to beat the inertia in carla physics.

**Comment by paulaurel** [comment]
@felipecode, I was running some experiments and lower throttle values of around 0.2 already beat the inertia.

**Comment by felipecode** [comment]
Thanks for the question BTW.
@paulaurel Yeah, but it goes very very slow right ? On 0.5 it like already input some constant speed. 
Inertia was not the right term. I haven't analyzed that properly to be fair. 
But there should be sometimes were the throttle is zero right ?

**Comment by paulaurel** [comment]
@felipecode yes it is relatively slow. It reaches approximately 14 km/h. Do you have a model to map the throttle values to corresponding acceleration? If not is there a model of the Carla's car drivetrain? This would be great.

**Comment by AliBaheri** [comment]
@felipecode Thanks for clarification.

**Comment by HubFire** [comment]
Why is there  negative numbers of speeds in the data set?
             
           speed        command          steer             throttle              brake
count  657600.000000  657600.000000  657600.000000  657600.000000  657600.000000
mean       17.935953       3.356726       0.001634       0.504136       0.174912
std        14.463256       1.173195       0.155913       0.330658       0.379184
**min       -18.739027**       0.000000      -1.154329       0.000000       0.000000
25%        10.102288       2.000000      -0.025604       0.500000       0.000000
50%        18.982355       3.000000      -0.000035       0.500000       0.000000
75%        22.477591       4.000000       0.029197       0.531509       0.000000
max        82.729424       5.000000       1.198919       1.000000       1.000000
